### PR TITLE
Disable storage and checkpoint store modules

### PR DESCRIPTION
**Pull request by srnagar**
Adding all the dependent modules to EH build is causing OOM in CI pipeline build. This change is to temporarily disable checkpoint store plugin module and it's storage dependencies to allow for event hubs release. 

Checkpoint store will be re-enabled after Storage Blobs preview.3 is released.

### CODE REVIEW

File: sdk/eventhubs/pom.service.xml [hunk 1]
```diff
@@ -34,12 +34,12 @@
         <module>../core/azure-core</module>
         <module>../core/azure-core-test</module>
         <module>../core/azure-core-amqp</module>
-        <module>../core/azure-core-http-netty</module>
-        <module>../identity/azure-identity</module>
-        <module>../storage/azure-storage-common</module>
-        <module>../storage/azure-storage-blob</module>
+<!--        <module>../core/azure-core-http-netty</module>-->
```
Comment (joshfree) [inline sdk/eventhubs/pom.service.xml]: Please add a tracking bug # to the

**Comment by srnagar** [comment]
No longer required.

**Pull request closed without merging**